### ISSUE TITLE
Same error is in the logs several times

### ISSUE THREAD

**Issue by mgledi** [argoproj/argo-rollouts#3006]
**Describe the bug**

We configured argo rollouts' logformat to be json. 
```
        chart_values:
          controller:
            extraArgs:
              - "--logformat"
              - "json"
              - "--loglevel"
              - "warn"
```

**To Reproduce**

Configure logformat=json

**Expected behavior**

All messages are in json format

**Screenshots**

**Version**

application version 1.5.1
chart version 2.31.5

**Logs**

```
{"generation":1,"level":"error","msg":"roCtx.reconcile err Operation cannot be fulfilled on replicasets.apps \"tm-tm-upgrade-dev-7d5c95ddd7\": the object has been modified; please apply your changes to the latest version and try again","namespace":"tm-tm-upgrade-dev","resourceVersion":"150709083","rollout":"tm-tm-upgrade-dev","time":"2023-09-01T08:41:48Z"}
{"level":"error","msg":"rollout syncHandler error: Operation cannot be fulfilled on replicasets.apps \"tm-tm-upgrade-dev-7d5c95ddd7\": the object has been modified; please apply your changes to the latest version and try again","namespace":"tm-tm-upgrade-dev","rollout":"tm-tm-upgrade-dev","time":"2023-09-01T08:41:48Z"}
time="2023-09-01T08:41:48Z" level=error msg="Operation cannot be fulfilled on replicasets.apps \"tm-tm-upgrade-dev-7d5c95ddd7\": the object has been modified; please apply your changes to the latest version and try again\n" error="<nil>"
...
time="2023-09-01T08:46:29Z" level=info msg="unknown field \"spec.template.metadata.creationTimestamp\"\n"
time="2023-09-01T08:46:29Z" level=info msg="unknown field \"spec.template.metadata.creationTimestamp\"\n"
time="2023-09-01T08:46:36Z" level=info msg="unknown field \"spec.template.metadata.creationTimestamp\"\n"
time="2023-09-01T08:46:36Z" level=info msg="unknown field \"spec.template.metadata.creationTimestamp\"\n"

```

---

**Message from the maintainers**:

Impacted by this bug? Give it a 👍. We prioritize the issues with the most 👍.

**Comment by zachaller** [comment]
This should be fixed in 1.6 https://github.com/argoproj/argo-rollouts/issues/2707

**Comment by zachaller** [comment]
Going to mark as close feel free to re-open if it's not fixed in 1.6
